automation_tools/__init__.py

```diff
@@ -628,6 +628,37 @@ def setup_abrt():
     run('abrt-auto-reporting enabled')
 
 
+def setup_code_coverage():
+    """Task to setup code coverage on sat6.
+
+    Expects the following environment variables:
+
+        SITECUSTOMIZE_URL
+                The sitecustomize file to launch the coverage.
+        COVERAGERC_URL
+                The coverage configuration file.
+
+    """
+    coverage_file = '/etc/coverage/.coveragerc'
```

Comment:
Since this is a constant, perhaps rename it to `COVERAGE_FILE` and put it together with other constants?